src/attributes/attr.js

```diff
@@ -34,7 +34,7 @@ jQuery.extend( {
 		}
 
 		// Fallback to prop when attributes are not supported
-		if ( typeof elem.getAttribute === "undefined" ) {
+		if ( elem.getAttribute === undefined ) {
```

Comment:
It was only certain properties in IE8, but I think we're safe now. We could probably do just...

```js
if ( !elem.getAttribute ) {
```